src/nvim/terminal.c

```diff
@@ -618,6 +621,17 @@ static int term_movecursor(VTermPos new, VTermPos old, int visible,
   return 1;
 }
 
+// Set the terminal title. Takes ownership of the title string.
```

Comment:
Follow the use of `value` (our title) in `dict_set_value()`, call path would be
```c
dict_set_value() ->
object_to_vim() , line 149 in helpers.c ->
xmemdupz(), line 614 in helpers.c 
```
There is no other use of `value` anywhere.